### PR TITLE
[tests-only] Match step to check group in the share response

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Match the these steps:
- `Then group "grp2" should be included in the response`
- `Then group "grp1" should not be included in the response`

## Related Issue
- Required for https://github.com/owncloud/ocis/pull/5076

## How Has This Been Tested?
- test environment: local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 1]
```diff
@@ -1431,15 +1431,18 @@ public function checkSharedFileAsPathNotInResponse(string $filename):void {
...
 	 *
+	 * @param string $type
 	 * @param string $user
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function checkSharedUserInResponse(string $user):void {
-		$user = $this->getActualUsername($user);
+	public function checkSharedUserInResponse(string $type, string $user):void {
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/Sharing.php:1442]:
```suggestion
	public function checkSharedUserOrGroupInResponse(string $type, string $userOrGroup):void {
```

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 2]
```diff
@@ -1448,16 +1451,17 @@ public function checkSharedUserInResponse(string $user):void {
...
+	 * @Then /^group "([^"]*)" should not be included in the response$/
 	 *
-	 * @param string $user
+	 * @param string $userOrGroup
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function checkSharedUserNotInResponse(string $user):void {
+	public function checkSharedUserNotInResponse(string $userOrGroup):void {
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/Sharing.php:1461]:
```suggestion
	public function checkSharedUserOrGroupNotInResponse(string $userOrGroup):void {
```

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]
Note: if there are tests that have both a user and a group with the same name, then this `Then` step might not be specific enough. In that case we might expect 2 entries in the response, one for user "xyz" and one for group "xyz". That will need a way to explicitly check for a combination of share_type and share_with.

**Pull request merged**